deployment/system-config.md

```diff
@@ -100,6 +100,20 @@ Logs event verbosely.
 
 Invokes fluentd without input plugins.
 
+### `with_source_only`
+
+| type | default | version |
+| :--- | :--- | :--- |
+| bool | nil | 1.18.0 |
+
+Not supported on Windows.
+
+Invokes a fluentd only with input plugins.
+
+See [Source Only Mode](source-only-mode.md) for details.
+
+See also [source_only_buffer section](#less-than-source_only_buffer-greater-than-section).
```

Comment:
It it intended for `#sourceonlybuffer-section`

Reply:
https://github.com/fluent/fluentd-docs-gitbook/pull/527#discussion_r1914534584

> Thanks, but behavior may be different from the production environment.
> 
> At least, the anchor would work because the following works now.
> 
>  * https://docs.fluentd.org/output/file#less-than-buffer-greater-than-section
>  * https://github.com/fluent/fluentd-docs-gitbook/blob/2483910bbf16c723d052b38d92b7e43e2bfce2e7/output/file.md#L28

Reply:
Oh, I've overlooked....